src/components/headings/H1.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/variables';
+
+h1.bdl-H1 {
```

Comment:
Is the `h1` specificity needed? That makes a modifier class require equivalent specificity to override any of the default properties